macOS/DuckDuckGo/WebExtensions/WebExtensionManager.swift

```diff
@@ -103,12 +105,26 @@ final class WebExtensionManager: NSObject, WebExtensionManaging {
         pathsCache.cache
     }
 
-    func addExtension(path: String) {
+    func addExtension(path: String) async {
         pathsCache.add(path)
+
+        do {
+            try await loader.loadWebExtension(path: path, into: controller)
+        } catch {
+            // This is temporary.  The actual handling of this error should be done outside of this manager.
+            assertionFailure("Failed to unload web extension \(path): \(error)")
```

Comment:
I'm drawing a line here - we should really just throw and let the parent handle the error, but I need to limit the scope of my changes at some point.